tests/hypervolume_tests/test_wfg.py

```diff
@@ -4,56 +4,43 @@
 import optuna
 
 
-def test_wfg_2d() -> None:
+def _shuffle_and_filter_sols(sols: np.ndarray, assume_pareto: bool) -> np.ndarray:
+    np.random.shuffle(sols)
```

Comment:
It would be great to specify the seed to ensure this test deterministic.

Reply:
Done